concent_api/core/validation.py

```diff
@@ -324,6 +324,12 @@ def validate_compute_task_def(compute_task_def: message.tasks.ComputeTaskDef) ->
                 ErrorCode.MESSAGE_INVALID
             )
 
+    if not extra_data.get('frames'):
+        raise ConcentValidationError(
+            "'extra_data' is missing in ComputeTaskDef",
```

Comment:
Please change the message, you have checked something different above.

Reply:
Done